app-engine/java/gradle/src/com/google/cloud/tools/intellij/appengine/java/gradle/AppEngineGradleModel.java

```diff
@@ -31,6 +31,9 @@
    */
   String gradleBuildDir();
 
-  /** Returns the path to the root of the Gradle module */
+  /** Returns the path to the root of the Gradle module. */
   String gradleModuleDir();
+
+  /** Returns the path to the web application directory. */
+  String webAppDir();
```

Comment:
in the class where this value is actually consumed (`AppEngineStandardGradleModuleComponent`) the `getWebAppDir` method returns an optionally wrapped value. 

if I were to make this return an optional, there would be a lot of wrapping and unwrapping boilerplate (since we should not be using optional parameters). wdyt?

Reply:
Yep, I saw that too - what if we just mark it `@Nullable` then as a reminder for later?

Reply:
sgtm. I'll add @Nullable to the other ones too for consistency

Reply:
done